common/source/docs/common-autopilots.rst

```diff
@@ -1,6 +1,57 @@
 .. _common-autopilots:
 
-==========================
+=====================
+Choosing an Autopilot
+=====================
+
+ArduPilot runs on many different autopilot boards, the most
+important of which are linked from the topic :ref:`AutoPilot Hardware Options <common-autopilots>`.
+
+Selecting the right board depends on the physical restraints of the
+vehicle, features desired, and the applications that you want to run.
+Factors to consider are:
+
+#. Sensor Redundancy: ArduPilot supports redundant IMUS, GPS, etc. Many controllers have multiple IMUs integrated on board.
```

Comment:
I'd add vibration isolation and IMU heater to this list of things to consider.

Reply:
historical, but will change